lib/DataSender.js

```diff
@@ -0,0 +1,14 @@
+const DataSender = {
```

Comment:
Or were you thinking of implementing the `Services.obs.notifyObservers` function call here?